roles/keystone-defaults/defaults/main.yml

```diff
@@ -83,6 +83,7 @@ keystone:
     sensu_checks:
       check_keystone_api:
         criticality: 'critical'
+    cron_bin_path:  "/sbin:/usr/sbin:{{ (os == 'rhel' ) | ternary('/usr/bin/', '/usr/local/bin/') }}"
```

Comment:
It seems that cron does not use the default PATH (I wonder if it's for security reasons?)
The job didn't run when I removed the PATH line. I'll put the path in the template for now.